### DISCUSSION TITLE
Inquiry about the tutorial

### DISCUSSION THREAD

**Discussion by keysongkang** [mzjb/DeepH-pack#51] [Q&A] [answered]
Dear DeepH community,

I am currently working on the tutorial for twisted bilayer bismuthene and following the provided instructions. However, I have noticed that my band structure exhibits some features but the overall shape does not agree with the result in the paper. I am unsure what might be causing this issue.

![image](https://github.com/mzjb/DeepH-pack/assets/130345610/d2e6c210-c47f-4e44-a310-f440fd1ee9e5)

Could you please provide some insights into the potential problem?

Thank you in advance for your assistance.

**Comment by mzjb** [comment, marked as answer]
Please see https://github.com/mzjb/DeepH-pack/issues/25#issuecomment-1437803433

**Reply by keysongkang** [reply]
Thank you for the link. I have two additional questions.

1. I still wonder why the shape of the band structure is different from the figure in the paper. I haven't changed any inputs.
2. When I resume the training process, is "resume" in the input enough? What is the difference between "pretrained" and "resume"? Do I have to set both inputs simultaneously?

**Reply by mzjb** [reply]
Based on the lengths between each tick mark on the x-axis of the graph you drew, it appears that there may be an issue with the selection of points K and M. Please try replacing K with (0.6666666666666667, 0.3333333333333333, 0) and replacing M with (0.5, 0.5, 0).

The correct proportions between the lengths of the ticks should be as shown in the following diagram: 
<img width="308" alt="image" src="https://github.com/mzjb/DeepH-pack/assets/39295563/8fea181b-72bf-4645-9911-5d5d12eb5ace">
